tracer/src/Datadog.Trace/AspNet/TracingHttpModule.cs

```diff
@@ -29,7 +29,7 @@ namespace Datadog.Trace.AspNet
     [EditorBrowsable(EditorBrowsableState.Never)]
     public class TracingHttpModule : IHttpModule
     {
-        internal static readonly IntegrationInfo IntegrationId = IntegrationRegistry.GetIntegrationInfo(nameof(IntegrationIds.AspNet));
+        internal static readonly IntegrationIds IntegrationId = IntegrationIds.AspNet;
```

Comment:
Can we rename the enum to `IntegrationId` (singular)?

> ✔️ DO use a singular type name for an enumeration unless its values are bit fields.

from https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations

Reply:
This had been bugging me too 😄 Changed in https://github.com/DataDog/dd-trace-dotnet/pull/2048/commits/ddbcbea66e6f38eb7dd461a9eefaf3fc0d989824